lib/manager/fvm/extract.ts

```diff
@@ -0,0 +1,39 @@
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { logger } from '../../logger';
+import type { PackageDependency, PackageFile } from '../types';
+
+interface FvmConfig {
+  flutterSdkVersion: string;
+}
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFile | null {
+  let fvmConfig: FvmConfig;
+  try {
+    fvmConfig = JSON.parse(content);
+  } catch (err) {
+    logger.debug({ packageFile }, 'Invalid FVM config');
+    return null;
+  }
+
+  if (!fvmConfig.flutterSdkVersion) {
+    logger.debug(
+      { contents: fvmConfig },
+      'FVM config does not have flutterSdkVersion specified'
+    );
+    return null;
+  } else if (!is.string(fvmConfig.flutterSdkVersion)) {
```

Comment:
Missing import for `is` 😉

Reply:
I blame GitHub for not being smart enough on auto commit 😛